index.html

```diff
@@ -12,18 +12,22 @@
     <section class="section-one">
         <header>
     
-            <div class="header-containt">
-    
-                <nav>
-                    <ul>
-                        <li><a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li><a href="#">Contact</a></li>
-                    </ul>
-    
-                </nav>
-    
-            </div>
+           <!-- Add a navigation icon -->
+<div class="nav-icon">
+    <i class="fas fa-bars"></i>
+  </div>
+  
+  <!-- Move the navigation list inside a new div -->
+  <div class="nav-list">
+    <nav>
+    <ul>
+      <li><a href="#">Portfolio</a></li>
+      <li><a href="#">About</a></li>
+      <li><a href="#">Contact</a></li>
+    </ul>
+    </nav>
+  </div>
+  
         </header>
         <section class="hero-comtain">
```

Comment:
Your hero container includes all the required elements, but there are some adjustments that still need to be made:

1. Font styles, line height, font family, font weight, and colors of the elements need to be adjusted to match the design. Please update your code accordingly to achieve a closer resemblance to the design.🙏 

2. The background image in the hero container does not match the design. Please make the necessary adjustments to ensure that the background image aligns with the intended design.🙏 

3. The left and right gaps between the elements and the screen are currently set to 0. It is important to follow the design and add the necessary gaps to create a visually pleasing layout. 🙏 

4. Additionally, there should be a down arrow button after the social icons. Please consider adding this button according to the design specifications.🙏 
![image](https://github.com/RolandAmini/portfolio-mobile/assets/55840999/46550f5d-5cae-4fe2-bc8f-bd212f7425c9)
5. Furthermore, the social icons should be represented as a list. Currently, you are using a `div` to display the icons. To ensure semantic correctness and improve accessibility, consider using a `ul` tag to group the icons and an `li` tag for each icon. You can also use `a` tags within the `li` tags to link your social networks to the icons using the `href` attribute.

**Yours:**
![image](https://github.com/RolandAmini/portfolio-mobile/assets/55840999/c9558e30-8740-4f90-9e5a-9f154031cd58)
**Design:**
![image](https://github.com/RolandAmini/portfolio-mobile/assets/55840999/57a3e5d0-f191-45fa-8623-31999bf360b3)

Please make these adjustments to enhance the visual appeal and functionality of your hero container. Great work so far, and keep up the good work!